### PR TITLE
f-header@2.0.0 beta.3: Navigation component

### PR DESCRIPTION
**Pull request by dandel10n**
### Added
- Navigation component
- `f-trak` attributes to the links

### Changed
- Resource files updated with the navigation links
- `@justeat/f-vue-icons` package updated to contain latest header icons

Styles for header itself will come in separate PR.
Tests for navigation will come later too.

<a href="https://gyazo.com/fef58479b5d1f951a5e56ff06334711f">Image from Gyazo</a>

### CODE REVIEW

**Review by simonsandersje — APPROVED** [review]

File: packages/f-header/src/tenants/da-DK.js [hunk 1]
```diff
@@ -2,14 +2,58 @@ export default {
...
+            text: 'Foretrukne kontaktmetode',
+            url: '/account/preferences',
+            gtm: 'click_account_contact_preferences'
+        }
+    },
+
+    logo: {
+        gtm: 'click_logo'
+    },
+    help: {
```
Comment (kevinrodrigues) [inline packages/f-header/src/tenants/da-DK.js:44]: Should there be line breaks here? Some have them and some don't.

File: packages/f-header/src/tenants/en-AU.js [hunk 2]
```diff
@@ -2,14 +2,48 @@ export default {
...
+            text: 'Delivery addresses',
+            url: '/account/addressbook',
+            gtm: 'click_account_address_books'
+        }
+    },
+
+    logo: {
+        gtm: 'click_logo'
+    },
+    help: {
```
Comment (kevinrodrigues) [inline packages/f-header/src/tenants/en-AU.js:34]: Similar to above.

**Review by kevinrodrigues — APPROVED** [review]
Will you be adding test support to these PRs?

File: packages/f-header/src/tenants/da-DK.js (hunk 1 above)
Comment (dandel10n) [reply packages/f-header/src/tenants/da-DK.js:44]: I just wanted to highlight navLinks a little bit with empty lines before and after for better visual reading the object

**Comment by dandel10n** [comment]
> Will you be adding test support to these PRs?

@kevinrodrigues  Yes, tests will come in separate PR, Sean is working on them at the moment.

I will merge this one and will add any improvements in my next PR, if there is more suggestions

**Pull request merged**